ome2024-ngff-challenge/src/About.svelte

```diff
@@ -10,57 +10,68 @@
+    that demonstrated the level of adoption of OME-Zarr and the relative ease of
+    conversion for some community members, but highlighted the often poor findability of the data
+    and the need for driving the OME-Zarr specification towards version 1.0.
+  <p>
+    A lofty goal of 1 Petabyte (PB) of data was set, mostly to make clear the scale of what <em>could be</em>.
   </p>
   <p>
     Work on the challenge started in earnest in July 2024. All progress is
     tracked in the <a href="https://github.com/ome/ome2024-ngff-challenge">ome2024-ngff-challenge</a>
-    repo. It was agreed that the results
-    of the OME-NGFF Challenge would be presented at the
+    repository. It was agreed that the deadline for submission would be such that the results
+    of the OME-NGFF Challenge could be presented at the
     <a href="https://globalbioimaging.org/exchange-of-experience/exchange-of-experience-ix">
       2024 Global BioImaging Meeting</a>.
   </p>
 
   <h2>Process</h2>
   <p>
-    The Challenge was run via a series of virtual meetings coordinated on the
+    The Challenge was run via a series of virtual meetings open to all which were coordinated on the
     Image.sc Forums, with all
     <a href="https://hackmd.io/3emKqKQsT_2U35vLepzDEQ">notes and lists of participants</a>
-    available.
+    available. Initial meetings including that at OME2024 focused on defining the scope of
+    changes that would be made to the OME-Zarr format. A Python-based tool to convert data
+    to this format was built. Submissions were then collected in simple CSV files with a Zarr
+    URL per row.
   </p>
 
   <h2>Results</h2>
   <p>
-    At the outset, we weren’t sure how much OME-Zarr data was available, if it
-    could be converted to Zarr V3, or how many organisations would participate.
-    In the end, we have far exceeded our expectations, with more than 0.5 PByte
-    of OME-Zarr made available, across a wide range of modalities.
+    At the outset, it was unclear how much OME-Zarr data was available, if it
+    could be converted to Zarr V3, and how many organisations would participate.
+    In the end, we are pleased with the more than 500 Terabytes of data which
+    have been made available, across a wide range of modalities.
   </p>
   <p>
-    To make the assembly as accessible as possible, we built the the OME-NGFF
-    Challenge Viewer [URL?}, which incorporates a CSV with locations of
-    datasets, and connects datasets to the OME NGFF Validator, for metadata
-    validation and viewing.
+    To make the collection as accessible as possible, the
+  <a href="https://ome.github.io/ome2024-ngff-challenge">OME-NGFF Challenge Viewer</a>,
+    was built which uses a CSV with locations of
+    datasets to enable collection statistics, browsing of thumbnails, and opening
+    the datasets with the OME NGFF Validator, for metadata
+    validation and viewing. Please let us know on image.sc or GitHub if you have any issues
+    or ideas.
```

Comment:
This one sentence is a bit too long. How about this.. (wasn't sure about "collection of statistics"). The sentence in [] is optional?

...was built, which loads the CSV files from contributors and combines all the Zarr data into a single collection. Data can be searched by key-word, filtered and sorted by various metadata and browsed with thumbnails [generated on the fly]. Links allow opening of datasets with the OME NGFF Validator for metadata validation and viewing.

Reply:
Thanks integrated with some updates if you want to review.